### DISCUSSION TITLE
My experience installing Neolink for the 4K RLC-810A

### DISCUSSION THREAD

**Discussion by joseortiz3** [thirtythreeforty/neolink#358] [General]
Before I forget, I just wanted to tell you about my experience I just had successfully using neolink with the RLC-810A 4K cameras for Blue Iris.

The install went OK. 
- I used chocolatey to install gstreamer as I already had it on my machine and trusted it more to set the environment variables. Install guide should probably explain more about using chocolatey and recommend it.
- I didn't know that IP addresses starting in 127 are loopback addresses, so I thought it was a typo in the installation guide and I used a 192 IP address, so I'll probably need to change that. Maybe explain this in the install guide, as the pictures show 192 and the text says 127. 

I got Neolink working with the RLC-810A which "supports" RTSP, so Neolink does work on (some) RTSP cameras. Why did I do this? The RTSP stream wasn't working for me from the camera. BlueIris struggled to decode it, freezing for 3-30 seconds repeatedly. I don't know why, and I am still working with Reolink and Blue Iris support to figure it out.

In contrast, Neolink works perfectly. The video is very smooth and the status panel in Blue Iris shows the feeds are consistent and strong. The task schedular starts neolink reliably (although I had to disable "check for network before starting") so I don't have to think about it anymore.

Thanks to Neolink, I am no longer dependent on the Blue Iris and Reolink support teams to help me get these cameras working. Maybe they will figure out how to fix my cameras' RTSP streams, but if they don't, I'm fine. If there's anything I've learned, it's that Reolink is shooting themselves in the foot by relying on their proprietary protocol.

**Comment by QuantumEntangledAndy** [comment]
Nice to hear it's working for well for you. Kind of keeps me motivated to hear this.

If you get any issues drop me a message and I'll see what we can do. We are currently working on improving the pause modes to save battery life of cameras.
